store/errors.go

```diff
@@ -49,6 +49,12 @@ var (
 
 	// ErrInvalidCredentials is returned on login error
 	ErrInvalidCredentials = errors.New("invalid credentials")
+
+	// ErrInvalidCredentials is returned when the user has not accepted the store's terms of service.
+	ErrTosNotAccepted = errors.New("terms of service not accepted")
+
+	// ErrInvalidCredentials is returned when the user has no valid payment methods associated with their account.
+	ErrNoValidPaymentMethod = errors.New("no valid payment methods")
```

Comment:
The point was to be clear it doesn't know the difference between never having added a payment method, or a credit card having expired. In that case they do have a payment method, it's just not valid.